.mailmap

```diff
@@ -94,8 +97,9 @@ Danny Nemer <hi@dannynemer.com> <DannyNemer@users.noreply.github.com>
 Danny Nemer <hi@dannynemer.com> <hi@DannyNemer.com>
 Dave Pacheco <dap@joyent.com> <dap@cs.brown.edu>
 David Cai <davidcai1993@yahoo.com> DavidCai <davidcai1993@yahoo.com>
-David Mark Clements <david.clements@nearform.com> davidmarkclements <david.clements@nearform.com>
-David Mark Clements <david.clements@nearform.com> davidmarkclements <huperekchuno@googlemail.com>
+David Mark Clements <david.clements@nearform.com>
+David Mark Clements <david.clements@nearform.com> <huperekchuno@googlemail.com>
+David Mark Clements <david.clements@nearform.com> <david.mark.clements@gmail.com>
```

Comment:
@davidmarkclements Just confirming, since you’ve been freshly listed with a third email address thanks to b1ef279d5726905d7941f4a58978b379daa3cdc4 – the NearForm one is still the one you use for `AUTHORS`?